languages/en/thingtalk.genie

```diff
@@ -441,6 +441,14 @@ $root = {
             | ('tell' | 'command' | 'order' | 'request' | 'ask') principal:constant_Entity__tt__username 'to send me' table:complete_table
         ) => C.makeProgram(new Ast.Statement.Command(table, [C.notifyAction('return')]), principal);
         ('tell' | 'command' | 'order' | 'request' | 'ask') principal:constant_Entity__tt__username 'to' ('let me know' | 'inform me' | 'notify me' | 'alert me') stream:stream => C.makeProgram(new Ast.Statement.Rule(stream, [C.notifyAction('return')]), principal);
+
+        ?extended_timers {
+            (
+                ('tell' | 'command' | 'order' | 'request' | 'ask') principal:constant_Entity__tt__username 'to' action:thingpedia_action stream:timer
+                | ('tell' | 'command' | 'order' | 'request' | 'inform') principal:constant_Entity__tt__username 'that' ('he needs' | 'she needs' | 'i need him' | 'i need her') 'to' action:thingpedia_action stream:timer
+            ) if complete => C.makeProgram(new Ast.Statement.Rule(stream, [action]), principal);
+            ('tell' | 'command' | 'order' | 'request' | 'ask') principal:constant_Entity__tt__username 'to' ('let me know' | 'inform me' | 'notify me' | 'alert me') stream:timer => C.makeProgram(new Ast.Statement.Rule(stream, [C.notifyAction('return')]), principal);
```

Comment:
the last one doesn't quite make sense: why would you go through a remote command to execute just the timer?

Reply:
You mean the one at line 450 right? Hmm we were wondering if this will allow both the sender and recipient to receive regular alerts? But I guess that's stretching it. Okay we'll drop that.